server/bin/gold/test-27.txt

```diff
@@ -50,9 +50,9 @@ len(actions) = 1
             "chunk_id": 1,
             "doctype": "status",
             "name": "pbench-audit-server",
-            "text": "pbench-audit-server.run-1970-01-01T00:00:42-UTC(unit-test)\n\n\nstart-1970-01-01T00:00:42-UTC: incoming hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/incoming\n\nControllers which do not have a /var/tmp/pbench-test-server/test-27/pbench/archive/fs-version-001 directory:\n\tcontroller-bad-names\n\nControllers which are empty:\n\tcontroller-prefixes\n\nend-1970-01-01T00:00:42-UTC: incoming hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/incoming\n\n\nstart-1970-01-01T00:00:42-UTC: results hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/results\n\nResults issues for controller: controller-prefixes\n\tEmpty tar ball directories:\n\t\tpath0/path1\n\t\tpre0/pre1/pre2\n\nend-1970-01-01T00:00:42-UTC: results hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/results\n\n\nstart-1970-01-01T00:00:42-UTC: users hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/users\n\nResults issues for controller: userA/controller-prefixes\n\tEmpty tar ball directories:\n\t\tpath0/path1\n\nend-1970-01-01T00:00:42-UTC: users hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/users\n",
+            "text": "pbench-audit-server.run-1970-01-01T00:00:42-UTC (unit-test)\n\n\n\nstart-1970-01-01T00:00:42-UTC: incoming hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/incoming\n\nControllers which do not have a /var/tmp/pbench-test-server/test-27/pbench/archive/fs-version-001 directory:\n\tcontroller-bad-names\n\nControllers which are empty:\n\tcontroller-prefixes\n\nend-1970-01-01T00:00:42-UTC: incoming hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/incoming\n\n\nstart-1970-01-01T00:00:42-UTC: results hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/results\n\nResults issues for controller: controller-prefixes\n\tEmpty tar ball directories:\n\t\tpath0/path1\n\t\tpre0/pre1/pre2\n\nend-1970-01-01T00:00:42-UTC: results hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/results\n\n\nstart-1970-01-01T00:00:42-UTC: users hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/users\n\nResults issues for controller: userA/controller-prefixes\n\tEmpty tar ball directories:\n\t\tpath0/path1\n\nend-1970-01-01T00:00:42-UTC: users hierarchy: /var/tmp/pbench-test-server/test-27/pbench/public_html/users\n",
             "total_chunks": 1,
-            "total_size": 1132
+            "total_size": 1134
```

Comment:
Why is the total size two bytes bigger (here and in other tests)? I've been going blind trying to discern the difference.

Reply:
Gah - GitHub does not flag it: three vs four newlines between `(unit-test)` and `start ...` on the first line

Reply:
I believe that GitHub is using Git which is using Diff which does not, by default, highlight within-a-line changes when the lines (or "difference regions") get very large.

I picked up on the pattern by looking at the smaller diffs with the highlights, and then I was able to find them in the long lines...but, yeah, it was a little crazy-making.